### DISCUSSION TITLE
Apprise Issue / Notifications Telegram

### DISCUSSION THREAD

**Discussion by MueJosh** [Nachtzuster/BirdNET-Pi#23] [Q&A] [answered]
Hey,

Yesterday I "Updated" my Birdnet-Pi (the original) to this Fork. (complete new install)

I adapted all my old Settings to this fork.
Now my Telegram Notificantions dont work anymore. If i send a test message it sends the last detections but it doesnt work on its own.
Any Ideas why? 

Any Code needed?

Thanks!
![birdnetpi_screenshot_Apprise-Issue](https://github.com/Nachtzuster/BirdNET-Pi/assets/145150281/0d225a5b-60d2-4ab0-b829-962941411679)

2. Another Issue but not problematic:
Updating the CONFIDENCE=XX crashes the Tools bar- it updates the new Confidence but i always have to open the "Tools"-Bar again. No Idea why...

**Comment by arne1921KF** [comment, marked as answer]
It's the percentage glyph `%` which is the culprit here. Noticed the same behaviour, checked the log and saw an error message.

**Reply by MueJosh** [reply]
That's it... Thank you!

**Reply by MueJosh** [reply]
Its fixed now, % works 👌
